vita3k/gui/src/home_screen.cpp

```diff
@@ -257,6 +257,8 @@ static bool app_filter(const std::string &app) {
         if (!filter_app_region({ "PCS" }))
             return true;
         break;
+    case ALL:
+        return true;
```

Comment:
remove it, is not needed, and is wrong, here return true want said skip game, one this case you skepp so all app
![image](https://user-images.githubusercontent.com/5261759/173254183-b0c801ec-d10b-4a3e-a032-a733da7a2cce.png)